server/crashmanager/templates/signatures/edit.html

```diff
@@ -4,7 +4,7 @@
 <createoredit
     {% if bucketId != None %} :bucket-id="{{ bucketId }}" {% endif %}
     {% if proposedSig != None %} :proposed-signature="{{ proposedSig }}" {% endif %}
-    {% if proposedDesc != None %} :proposed-description="'{{ proposedDesc }}'" {% endif %}
-    {% if warningMessage != None %} :warning-message="'{{ warningMessage }}'" {% endif %}
+    {% if proposedDesc != None %} proposed-description="{{ proposedDesc }}" {% endif %}
+    {% if warningMessage != None %} warning-message="{{ warningMessage }}" {% endif %}
```

Comment:
Will this automatically escape quotations in the attr value?  Or will this still fail if the attr value contains a double quote?

Reply:
Confirmed this is escaped, and I can create a signature using the crash originally used in #757